src/js/index.js

```diff
@@ -1,12 +1,31 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var makeSubmitForm = require('./makeSubmitForm.jsx');
+var api = require('./api');
+var userSelect = require('./userSelect.jsx');
+var fiContainer = require('./fiContainer.jsx');
+//var makeSubmitForm = require('./makeSubmitForm.jsx');
 
+/*
 var SubmitForm = makeSubmitForm('/submit', function(e){
   console.log(e, 'file uploaded');
 });
+*/
 
 ReactDOM.render(
```

Comment:
Sorry, meant that on the userSelect stuff. We wouldn't have a dropdown to choose the user there.